documentation/staging/content/faq/external-clients.md

```diff
@@ -366,6 +381,129 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
+                MinDynamicClusterSize: '0'
+                CalculatedListenPorts: false
+            DefaultLoadAlgorithm: 'round-robin-affinity'
+```
+
+#### Configuring External Listen Address for WebLogic Default Channel
+
+WebLogic Server provides an External Listen Address/DNS Name for use in RMI stubs to allow external clients to connect to the server through a Kubernetes external service name. For more information on 'External Listen Address' (i.e. ExternalDNSName attribute) on the ServerMBean, see ExternalDNSName attribute in [ ServerMBean reference ](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/wlmbr/core/index.html).
+
+Here are steps for setting the 'ExternalDNSName' attribute on the ServerMBean, for the default channel, for a domain source type __Domain in PV__:
+
+```
+How to set the 'External Listen Address' on the ServerMBean for stand-alone server 'AdminServer': 
+=========================================== 
+
+1. Go to Admin console 
+2. Expand the Environment link on the left panel 
+3. Click on the Servers link on the sub-panel
+4. On the right panel, click on AdminServer -> Configuration -> General tabs. 
+5. Click on the Advanced link to expand the server configuration page.
+6. Enter the service name for the WLS server pod in the External Listen Address field.
+   For example:
+     i. For Admin Server the service name would be of the form <DOMAIN_UID>-admin-server.<NAMESPACE> 
+     ii. For a managed server the service name would be of the form <DOMAIN_UID>-managed-server${id}.<NAMESPACE>
+``` 
+
+Here are steps for setting the 'ExternalDNSName' attribute on the ServerTemplateMBean, for the default channel, for a domain source type __Domain in PV__:
+
+```
+How to set the 'External Listen Address' on the ServerTemplateMBean for server template named 'cluster-1-template': 
+=========================================== 
+
+1. Go to Admin console 
+2. Expand the Environment link on the left panel 
+3. Click on the Clusters link on the sub-panel
+4. Click on the Server Templates link on the sub-panel
+5. On the right panel, click on cluster-1-template -> Configuration -> General tabs. 
+6. Click on the Advanced link to expand the server configuration page.
+7. Enter the service name for the WLS server pod in the External Listen Address field.
+   For example:
+     i. For Admin Server the service name would be of the form <DOMAIN_UID>-admin-server.<NAMESPACE> 
+     ii. For a managed server the service name would be of the form <DOMAIN_UID>-managed-server${id}.<NAMESPACE>
+``` 
+
+Here is a snippet of offline WLST code for setting the 'External Listen Address' for a stand-alone server named 'AdminServer':
+
+```javascript
+serverName = "AdminServer"
+cd('/Servers/%s' % serverName)
+set('ExternalDNSName', 'domain1-adminserver.weblogic-domain')
+```
+
+Here is a snippet of offline WLST code for setting the 'External Listen Address' for a server template named 'cluster-1-template':
```

Comment:
Please update intro sentence to mention namespace and domain-uid and cluster-name, update the script to set vars for same, update the script to calc DNS1123 name, etc.

Reply:
Also update the ExternalDNSName to use the managed-server${id} instead of managed-server1

Reply:
(WebLogic itself will substitute the right value for ${id} at runtime.)